tests/integration/security/authorization_test.go

```diff
@@ -906,54 +831,114 @@ func TestAuthorization_TCP(t *testing.T) {
 					ExpectAllowed: expectAllowed,
 				}
 			}
+			ns := apps.Namespace1
+			ns2 := apps.Namespace2
+			src0 := apps.A.Match(echo.Namespace(ns.Name()))
+			src1 := apps.B.Match(echo.Namespace(ns.Name()))
+			src2 := apps.E.Match(echo.Namespace(ns.Name()))
+			src3 := apps.C.Match(echo.Namespace(ns2.Name()))
+
+			dst0 := apps.B.Match(echo.Namespace(ns.Name()))
+			dst1 := apps.C.Match(echo.Namespace(ns.Name()))
+			dst2 := apps.D.Match(echo.Namespace(ns.Name()))
+			dst3 := apps.E.Match(echo.Namespace(ns.Name()))
+			dst4 := apps.A.Match(echo.Namespace(ns.Name()))
+			t.NewSubTest(fmt.Sprintf("to %s %s %s %s %s", dst0[0].Config().Service,
```

Comment:
this is extremely unreadable, in the log I just see `=== RUN   TestAuthorization_TCP/to_b_c_d_e_a/a->b:http-8091/data[deny]` which I have no idea what does the `b_c_d_e_a` mean in the name, could you improve to make it more readable here?